homeassistant/components/ping/binary_sensor.py

```diff
@@ -156,7 +185,7 @@ def ping(self):
         except (subprocess.CalledProcessError, AttributeError):
```

Comment:
```suggestion
        except AttributeError:
```

Reply:
done

Reply:
I got this error after some time. Any ideas on how to fix it?

> 2020-08-17 13:22:06 ERROR (MainThread) [homeassistant.components.ping.binary_sensor] Timed out running command: `['ping', '-n', '-q', '-c', '5', '-W1', '212.77.98.9']`, after: 8s
> Traceback (most recent call last):
>   File "/workspaces/home-assistant/homeassistant/components/ping/binary_sensor.py", line 143, in async_ping
>     out_data, out_error = await asyncio.wait_for(
>   File "/usr/local/lib/python3.8/asyncio/tasks.py", line 490, in wait_for
>     raise exceptions.TimeoutError()
> asyncio.exceptions.TimeoutError
> 2020-08-17 13:22:06 ERROR (MainThread) [homeassistant.helpers.entity] Update for binary_sensor.wp_pl fails
> Traceback (most recent call last):
>   File "/workspaces/home-assistant/homeassistant/components/ping/binary_sensor.py", line 143, in async_ping
>     out_data, out_error = await asyncio.wait_for(
>   File "/usr/local/lib/python3.8/asyncio/tasks.py", line 490, in wait_for
>     raise exceptions.TimeoutError()
> asyncio.exceptions.TimeoutError
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/workspaces/home-assistant/homeassistant/helpers/entity.py", line 263, in async_update_ha_state
>     await self.async_device_update()
>   File "/workspaces/home-assistant/homeassistant/helpers/entity.py", line 454, in async_device_update
>     await self.async_update()  # type: ignore
>   File "/workspaces/home-assistant/homeassistant/components/ping/binary_sensor.py", line 101, in async_update
>     await self._ping.async_update()
>   File "/workspaces/home-assistant/homeassistant/components/ping/binary_sensor.py", line 199, in async_update
>     self.data = await self.async_ping()
>   File "/workspaces/home-assistant/homeassistant/components/ping/binary_sensor.py", line 188, in async_ping
>     await pinger.kill()
>   File "/usr/local/lib/python3.8/asyncio/subprocess.py", line 144, in kill
>     self._transport.kill()
>   File "/usr/local/lib/python3.8/asyncio/base_subprocess.py", line 153, in kill
>     self._check_proc()
>   File "/usr/local/lib/python3.8/asyncio/base_subprocess.py", line 142, in _check_proc
>     raise ProcessLookupError()
> ProcessLookupError

Reply:
@bdraco is this error correct or should I handle it in some way?